website/docs/r/vapp_access_control.html.markdown

```diff
@@ -0,0 +1,135 @@
+---
+layout: "vcd"
+page_title: "vCloudDirector: vcd_vapp_access_control"
+sidebar_current: "docs-vcd-resource-vapp-access-control"
+description: |-
+  Provides a vCloud Director Access Control structure for a vApp.
+---
+
+# vcd\_vapp\_access\_control
+
+Provides a vCloud Director Access Control structure for a vApp. This can be used to create, update, and delete access control structures for a vApp.
+
+!> **Warning:** The access control info is tied to a vApp. Thus, there could be only one instance per vApp. Using a different
+definition for the same vApp ID will result in a previous instance to be overwritten.
```

Comment:
Is there a programmatic way of imposing this 1:1 constraint?

Reply:
It's the same situation as for `vapp_nat_rules`, `vapp_static_routing`, and `vapp_firewall_rules`.
A vApp can have only one set of each of these resources.